frameworks/hdfs/tests/test_tls.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+
+import sdk_cmd
+import sdk_install
+import sdk_hosts
+import sdk_plan
+import sdk_security
+import sdk_utils
+import shakedown
+from tests import config
+from tests.config import (
+    DEFAULT_TASK_COUNT,
+    PACKAGE_NAME,
+    SERVICE_NAME,
+)
+
+
+DEFAULT_JOURNAL_NODE_TLS_PORT = 8481
+DEFAULT_NAME_NODE_TLS_PORT = 9003
+DEFAULT_DATA_NODE_TLS_PORT = 9006
+
+
+@pytest.fixture(scope='module')
+def service_account():
+    """
+    Creates service account with `hdfs` name and yields the name.
+    """
+    name = SERVICE_NAME
+    sdk_security.create_service_account(
+        service_account_name=name, service_account_secret=name)
+     # TODO(mh): Fine grained permissions needs to be addressed in DCOS-16475
```

Comment:
Since this function is used in all the TLS tests, we could consider moving it to a function in `sdk_security` (perhaps in a different PR).

Reply:
https://jira.mesosphere.com/browse/INFINITY-2419